glitter_news/mixins.py

```diff
@@ -1,8 +1,20 @@
 # -*- coding: utf-8 -*-
 
+from django.utils import six
+
 from .models import Post
 
 
 class PostMixin(object):
     def get_queryset(self):
-        return Post.objects.published()
+        queryset = Post.objects.published()
+
+        ordering = self.get_ordering()
+        if ordering:
+            if isinstance(ordering, six.string_types):
+                ordering = (ordering,)
+            queryset = queryset.order_by(*ordering)
+        return queryset
+
+    def get_ordering(self):
```

Comment:
Need more ordering fields, otherwise the list view won't come out as expected - as it's just ordering on one field.

eg.`('-is_sticky', '-date')`